### PR TITLE
[release-1.35] CVE-2024-1753 fix, bump to v1.35.1, then v1.35.2-dev

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # Changelog
 
+## v1.35.1 (2024-03-18)
+
+    [relase-1.35] CVE-2024-1753 container escape fix
```
Comment (mtrmac) [inline CHANGELOG.md:7]: @TomSweeneyRedHat Just to give you the choice: if you want to respin with this typo fixed (also in the commit message where it came from), please merge afterwards without another review.

Alternatively, feel free to drop the hold and let this through.

Comment (TomSweeneyRedHat) [reply CHANGELOG.md]: @mtrmac I completely missed that.  Thanks, fixed and repushed.

**Pull request merged**